substra/sdk/backends/local/compute/worker.py

```diff
@@ -72,30 +125,103 @@ def _context(self, key):
             # delete tuple working directory
             shutil.rmtree(tmp_dir, ignore_errors=True)
 
-    def schedule_traintuple(self, tuple_):
+    def schedule_composite_traintuple(self, tuple_):
```

Comment:
The schedule_composite_traintuple and schedule_traintuple are pretty similar and schedule_traintuple already handles both train and aggregatetuples. Do you think this could be refactored to have a single schedule_traintuple method that would handle all types?

Reply:
I can let's see how it looks, 
I am not entirely convinced because the 'out_models' attribute has a different name, the commands and volumes are different too, but the steps are indeed the same.

Reply:
Done, there are "ifs" at each step but it is actually easy to read I think